doc/release-notes-0.4.0.rst

```diff
@@ -0,0 +1,274 @@
+.. _ncs_release_notes_040:
+
+|NCS| v0.4.0 Release Notes
+##########################
+
+This project is hosted by Nordic Semiconductor to demonstrate the integration of Nordic SoC support in open source projects, like MCUBoot and the Zephyr RTOS, with libraries and source code for low-power wireless applications.
+
+nRF Connect SDK v0.4.0 supports development with nRF9160 Cellular IoT devices.
+It contains references and code for Bluetooth Low Energy devices in the nRF52 Series, though development on these devices is not currently supported with the nRF Connect SDK.
+
+Highlights
+**********
+
+* Added support for the new triple mode (LTE-M/NB-IoT/GPS) with ``mfw_nrf9160_0.7.0-29.alpha`` (see :ref:`nrf9160_ug_network_mode` for instructions for changing the network mode)
```

Comment:
The user guide explains only how to switch between LTE-M and NB-IoT, not GPS - should the link be included?

Reply:
Don't think that is strictly necessary, since next bullet point says there is a sample on how to use GPS.

Reply:
The AT command document also explains how the "AT%XSYSTEMMODE" and "AT%XMAGPIO" commands works. Maybe refer to this documentation, or do the nrf9160_ug_network_mode section already mention this document?

Reply:
Yes, the nrf9160_ug_network_mode section points to the system mode section in the AT commands documentation. So I guess this is the best we can do for now. :)